tests/phpstan/bootstrap.php

```diff
@@ -0,0 +1,67 @@
+<?php
```

Comment:
we don't need any phpstan config anymore. SwagExtensionStore needs to be required with composer and then the core phpstan can find it

Reply:
I could not get it running locally :thinking:

Reply:
I removed the PHPStan command from the composer.json.